crates/sui/tests/transaction_orchestrator_tests.rs

```diff
@@ -278,7 +278,7 @@ async fn test_tx_across_epoch_boundaries() {
     // Will all transactions finish in epoch 0?  Local testing results say
     // unlikely with total_tx_size = 180.
     loop {
-        match tokio::time::timeout(tokio::time::Duration::from_secs(5), result_rx.recv()).await {
+        match tokio::time::timeout(tokio::time::Duration::from_secs(60), result_rx.recv()).await {
```

Comment:
The test was previously succeeding in < 5 seconds as the transaction in question is a non-shared object one which didn't require consensus to be up and running - thus test could succeed as consensus might not be up and running in new epoch, but transaction would succeed. Since this PR is introducing Narwhal wait to shutdown before completing reconfiguration, it can take much more time to complete.